### PR TITLE
Feature/289 create a feedback page in our codebase

### PR DESCRIPTION
**Pull request by gazaston**
after

### CODE REVIEW

**Comment by gazaston** [comment]
[Create a feedback page in our codebase](https://trello.com/c/0r515G5g/289-create-a-feedback-page-in-our-codebase)

File: app/views/hiring_staff/vacancies/feedback/new.html.haml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+- content_for :page_title_prefix, 'Feedback'
+
+%h1.heading-large
+  Give feedback
```
Comment (despo) [inline app/views/hiring_staff/vacancies/feedback/new.html.haml]: extract to locale

File: app/views/hiring_staff/vacancies/feedback/new.html.haml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- if @feedback.errors.any?
+  #errors.error-summary{role: 'alert', tabindex: '-1'}
+    %h2.heading-medium.error-summary-heading
+      Please correct the following error
```
Comment (despo) [inline app/views/hiring_staff/vacancies/feedback/new.html.haml]: extract to locale

File: app/views/hiring_staff/vacancies/feedback/new.html.haml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    %ul.error-summary-list
+      - @feedback.errors.each do |attribute, error|
+        %li= @feedback.errors.full_message(attribute, error)
+
+= simple_form_for @feedback, action: :post, url: school_job_feedback_path do |f|
+  .grid-row
+    .column-two-thirds
+      .form-group
+        %legend
+          %h2.heading-medium Overall, how did you feel about the service you received today?
```
Comment (despo) [inline app/views/hiring_staff/vacancies/feedback/new.html.haml]: extract to locale

File: app/views/hiring_staff/vacancies/summary.html.haml [hunk 4]
```diff
@@ -1,21 +1,25 @@
-- content_for :page_title_prefix, "Your job posting for #{@school.name} has been submitted"
+- content_for :page_title_prefix, "The job listing for #{@school.name} has been completed"
```
Comment (despo) [inline app/views/hiring_staff/vacancies/summary.html.haml]: extract to locale

File: app/views/hiring_staff/vacancies/feedback/new.html.haml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        %fieldset
+          = f.input :rating, as: :radio_buttons, collection: rating_options, label: false, required: true, item_wrapper_tag: :div, item_wrapper_class: 'multiple-choice'
+          = f.input :comment, as: :text, label: 'How could we improve this service?', hint: 'Limit is 200 characters. Please don\'t include any personal or financial information, for example your National Insurance or credit card numbers', input_html: { class: 'form-control form-control-4-4', rows: '5' }
```
Comment (despo) [inline app/views/hiring_staff/vacancies/feedback/new.html.haml]: extract to locale, both label and hint text

File: app/views/hiring_staff/vacancies/feedback/new.html.haml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        %li= @feedback.errors.full_message(attribute, error)
+
+= simple_form_for @feedback, action: :post, url: school_job_feedback_path do |f|
+  .grid-row
+    .column-two-thirds
+      .form-group
+        %legend
+          %h2.heading-medium Overall, how did you feel about the service you received today?
+        %fieldset
+          = f.input :rating, as: :radio_buttons, collection: rating_options, label: false, required: true, item_wrapper_tag: :div, item_wrapper_class: 'multiple-choice'
```
Comment (despo) [inline app/views/hiring_staff/vacancies/feedback/new.html.haml]: extract to locale, both label and hint text

File: app/views/hiring_staff/vacancies/feedback/new.html.haml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+          = f.input :comment, as: :text, label: 'How could we improve this service?', hint: 'Limit is 200 characters. Please don\'t include any personal or financial information, for example your National Insurance or credit card numbers', input_html: { class: 'form-control form-control-4-4', rows: '5' }
+          = f.submit 'Submit feedback', class: 'button'
```
Comment (despo) [inline app/views/hiring_staff/vacancies/feedback/new.html.haml]: extract to locale

File: app/views/hiring_staff/vacancies/summary.html.haml [hunk 8]
```diff
@@ -1,21 +1,25 @@
...
-      %h1.heading-large.mv0
+      %h1.heading-large.mt0
         = t('jobs.submitted')
 
+      - if @vacancy.publish_today?
+        %p= link_to t('jobs.preview_posted_job'), job_path(@vacancy), class: 'reversed-link'
+      - else
+        %p= link_to t('jobs.preview_posted_job'), school_job_path(@vacancy.id), class: 'reversed-link'
+
+      %p.mb0 You can also view, amend or delete it on #{link_to "your school's dashboard", new_sessions_path, class: 'reversed-link'}
```
Comment (despo) [inline app/views/hiring_staff/vacancies/summary.html.haml]: extract to locale

File: app/views/hiring_staff/vacancies/summary.html.haml [hunk 9]
```diff
@@ -1,21 +1,25 @@
...
-      %p
-        = t('jobs.view_posted_job')
-      %p
-        = link_to job_url(@vacancy), job_path(@vacancy)
-    - else
-      %p
-        = t('jobs.preview_posted_job', date: @vacancy.publish_on)
-      %p
-        = link_to school_job_url(@vacancy.id), school_job_path(@vacancy.id)
+    %h3.heading-medium What happens next
```
Comment (despo) [inline app/views/hiring_staff/vacancies/summary.html.haml]: Text to be extracted to locale

File: spec/features/hiring_staff_can_submit_vacancy_feedback_spec.rb [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+      scenario 'must have a rating specified' do
+        visit new_school_job_feedback_path(published_job.id)
+
+        fill_in 'feedback_comment', with: 'Perfect!'
+
+        click_on 'Submit feedback'
+        expect(page).to have_content(' Rating can\'t be blank')
+      end
+
+      scenario 'can not be accessed for non-published vacancies' do
```
Comment (despo) [inline spec/features/hiring_staff_can_submit_vacancy_feedback_spec.rb]: This was my mistake  🤣  
the scenario should change to `Can be successfuly submitted for a published vacancy`

Comment (despo) [reply spec/features/hiring_staff_can_submit_vacancy_feedback_spec.rb]: double l!

**Pull request merged**